clients/csharp-uwp/UWPVoiceAssistantSample/Settings/AppSettings.cs

```diff
@@ -107,52 +130,52 @@ public static bool IsValidAzureRegion(string region)
                 "westeurope",
                 "northeurope",
                 "southeastasia",
-            }.Contains(region, StringComparer.CurrentCultureIgnoreCase);
```

Comment:
Unrelated to your change, but we now have 3 new regions for DLS. See e-mail from Minko I forwarded you:
•	West Central US
•	East Asia
•	Central India

Reply:
Updated appsettings with the above regions